src/syft/lib/python/int.py

```diff
@@ -372,8 +372,21 @@ def conjugate(self) -> SyPrimitiveRet:
         res = super().conjugate()
         return PrimitiveFactory.generate_primitive(value=res)
 
-    def to_bytes(self, length: int, byteorder: str, signed: bool = True) -> bytes:
-        return super().to_bytes(length, byteorder, signed=signed)
+    # method signature override
+    def to_bytes(
+        self,
+        length: Optional[int] = None,
+        byteorder: Optional[str] = None,
+        signed: Optional[bool] = True,
+    ) -> bytes:
+        if length is not None and byteorder is not None and signed is not None:
```

Comment:
MAYBE:
Rewrite this to:
```if length and byteorder and signed is not None:```

Reply:
I guess two things:
1) are there values of length and byteorder that are valid but evaluate to False, like 0
2) would it be better to forward the incorrect input to the correct method so the error is meaningful, this code is already a bit suspect but but means that if any of those args or kwargs are passed in the original method will be called.

I think it would be better to avoid this by renaming again, but are we confident we wont come across a type that uses the new attr like "to_binary" again down the track. Perhaps we should prefix with sy?
```
a = Int(1)
a.sy_bytes()
```

Or now that we don't really have multiple encoding types perhaps we should just change this back to "serialize?"
```
a = Int(1)
a.serialize(to_bytes=True)
```

Reply:
Issue here: https://github.com/OpenMined/PySyft/issues/4759